nj2k/nj2k-services/src/org/jetbrains/kotlin/nj2k/postProcessing/J2kPostProcessor.kt

```diff
@@ -220,7 +220,15 @@ private val inspectionLikePostProcessingGroup =
         MayBeConstantInspectionBasedProcessing(),
         RemoveForExpressionLoopParameterTypeProcessing(),
         intentionBasedProcessing(ReplaceMapGetOrDefaultIntention()),
-        inspectionBasedProcessing(ReplaceGuardClauseWithFunctionCallInspection())
+        inspectionBasedProcessing(ReplaceGuardClauseWithFunctionCallInspection()),
+        intentionBasedProcessing(ConvertToRawStringTemplateIntention()) { element ->
+            element.parents.none {
```

Comment:
Why don't use `ConvertToStringTemplateIntention.shouldSuggestToConvert` here instead of custom check?

Reply:
`ConvertToStringTemplateIntention.shouldSuggestToConvert` returns false for:
```
var s = "asdf\n" +
        "nadfadsf\n" +
        "asdfasdf"
```

https://github.com/t-kameyama/kotlin/blob/fd53fe7fd1e5830ac7fcbe8127ce662c8c7d2b60/idea/src/org/jetbrains/kotlin/idea/intentions/ConvertToStringTemplateIntention.kt#L60
```entries.none { it is KtBlockStringTemplateEntry }``` returns false.